src/components/Comment/index.jsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+import comments from "../../data/comments"; 
+import CommentElement from "./CommentElement";
+
+const Comment = ({ postId }) => {
+    const [commentsList, setCommentsList] = useState(comments);
+    const [commentInputValue, setCommentInputValue] = useState("");
+    const [newId, setNewID] = useState(4);
```

Comment:
여기 4로 하드코딩하기보다는 comments에서 가져온 length를 초기값으로 설정해 보는 건 어떨까요
```suggestion
    const [newId, setNewID] = useState(comments.length);
```